agogosml/agogosml/common/kafka_streaming_client.py

```diff
@@ -147,7 +119,7 @@ def start_receiving(self, on_message_received_callback):
                 # Poll messages from topic
                 msg = self.read_single_message()
                 if msg is not None:
-                    on_message_received_callback(msg)
+                    on_message_received(msg)
```

Comment:
oof..

Reply:
This is a simple rename since the type annotation now already makes it explicit that it's a callback. No need to include it in the variable name.